packages/e2e-tests/helpers/source-panel.ts

```diff
@@ -366,28 +374,32 @@ export async function verifyLogPointContentTypeAheadSuggestions(
   }
 }
 
-export async function getSelectedLineNumber(page: Page): Promise<number | null> {
-  let currentLineLocator = null;
+export async function getSelectedLineNumber(
+  page: Page,
+  lineIsCurrentPoint: boolean
+): Promise<number | null> {
+  const highlightLineTestName = lineIsCurrentPoint
+    ? "CurrentExecutionPointLineHighlight"
+    : "ViewSourceHighlight";
```

Comment:
Ooo, this is a bit awkward huh.

I think maybe we should change the view to make tests easier to write:
```js
switch (mode) {
  case "search-result": {
    return (
      <div
        className={styles.CurrentSearchResult}
        data-test-name="CurrentLineHighlight"
        data-test-type="search-result"
      />
    );
  }
  case "view-source": {
    return (
      <div
        className={styles.ViewSourceHighlight}
        data-test-name="CurrentLineHighlight"
        data-test-type="view-source"
      />
    );
  }
}
```

What do you think?

Reply:
I think this will make the helper and component easier to write future tests for.